docs/pages/setup/guides.mdx

```diff
@@ -20,4 +20,7 @@ layout: tocless-doc
   <li>
     [Joining Nodes in AWS](./guides/joining-nodes-aws.mdx). How to join nodes and proxies on EC2.
   </li>
+  <li>
+    [Teleport with GitHub](./guides/ssh-key-extensions.mdx). How to use the Teleport certificate authority with GitHub organizations.
```

Comment:
Let's make the description the same as the `description` field of the frontmatter in the guide.